SpatialGDK/Source/SpatialGDK/Public/Utils/LockingStatics.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "GameFramework/Actor.h"
+#include "Kismet/BlueprintFunctionLibrary.h"
+
+#include "LockingStatics.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogLocking, Log, All);
+
+USTRUCT(BlueprintType)
+struct FLockingToken
+{
+	GENERATED_USTRUCT_BODY()
+
+	UPROPERTY(BlueprintReadOnly)
```

Comment:
I thought int64 blueprint properties errored?

Reply:
I can grab and print it okay 
![image](https://user-images.githubusercontent.com/9222722/88673846-fc28b780-d0e0-11ea-8437-09db273810b2.png)